packages/@lwc/engine-server/src/__tests__/fixtures/attribute-aria-modify/modules/x/cmp/cmp.js

```diff
@@ -0,0 +1,13 @@
+import { LightningElement } from 'lwc';
+
+export default class extends LightningElement {
+  connectedCallback() {
+    // Modify this component's aria properties at runtime, which should be reflected to attributes
+
+    // Standard ARIA property
+    this.ariaBusy = 'true'
+
+    // Non-standard LWC-specific legacy ARIA property
+    this.ariaActiveDescendant = 'foo'
```

Comment:
This is exactly what used to not work in SSR. The other SSR fixtures worked fine.